tests/framework/integration/cluster.go

```diff
@@ -1383,6 +1383,13 @@ func (c *Cluster) ClusterClient() (client *clientv3.Client, err error) {
 			DialTimeout: 5 * time.Second,
 			DialOptions: []grpc.DialOption{grpc.WithBlock()},
```

Comment:
Would be better to populate the following fields as well, FYI. [13690/files](https://github.com/etcd-io/etcd/pull/13690/files)
```
MaxCallSendMsgSize: c.Cfg.ClientMaxCallSendMsgSize,
MaxCallRecvMsgSize: c.Cfg.ClientMaxCallRecvMsgSize,
Logger:             memberLogger(t, "cluster"),
```

Reply:
Done